github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-postsubmits.yaml

```diff
@@ -47,8 +46,76 @@ postsubmits:
           - >
             cat $QUAY_PASSWORD | podman login --username $(<$QUAY_USER) --password-stdin quay.io &&
             ./publish.sh &&
+            SHORT_SHA=$(git rev-parse --short HEAD) &&
+            # Adds git tag created by publish.sh above into the file
+            echo "$(git tag --points-at HEAD | head -1)" > amd64-$SHORT_SHA &&
+            gsutil cp ./amd64-$SHORT_SHA gs://kubevirt-prow/release/kubevirt/kubevirtci/amd64-$SHORT_SHA
+          # docker-in-docker needs privileged mode
+          env:
+          - name: GIMME_GO_VERSION
+            value: "1.22.5"
+          securityContext:
+            privileged: true
+          volumeMounts:
+          - mountPath: /dev
+            name: devices
+          resources:
+            requests:
+              memory: "29Gi"
+    - name: publish-kubevirtci-s390x
+      branches:
+      - main
+      always_run: true
+      annotations:
+        testgrid-create-test-group: "false"
+      decorate: true
+      decoration_config:
+        timeout: 2h
+      max_concurrency: 1
+      extra_refs:
+      - org: kubevirt
+        repo: project-infra
+        base_ref: main
+      labels:
+        preset-podman-in-container-enabled: "true"
+        preset-docker-mirror-proxy: "true"
+        preset-gcs-credentials: "true"
+        preset-github-credentials: "true"
+        preset-kubevirtci-quay-credential: "true"
+      cluster: s390x-prow-virtual
+      spec:
+        volumes:
+        - hostPath:
+            path: /dev
+            type: Directory
+          name: devices
+        containers:
+        - image: quay.io/kubevirtci/golang:v20240814-9b6c4af
+          command:
+          - "/usr/local/bin/runner.sh"
+          - "/bin/bash"
+          - "-c"
+          - >
+            cat $QUAY_PASSWORD | podman login --username $(<$QUAY_USER) --password-stdin quay.io &&
+            SHORT_SHA=$(git rev-parse --short HEAD) &&
+            GCS_FILE_PATH=release/kubevirt/kubevirtci/amd64-$SHORT_SHA &&
+            source /usr/local/bin/gcs_restapi.sh
+            CHECK_INTERVAL=30 &&
+            while true; do
+                if stat_gcs_file kubevirt-prow "$GCS_FILE_PATH"; then
+                    echo "File $GCS_FILE_PATH is now available."
+                    break
+                else
+                    echo "File $GCS_FILE_PATH not found. Checking again in $CHECK_INTERVAL seconds."
+                    sleep $CHECK_INTERVAL
+                fi
+            done &&
+            export KUBEVIRTCI_TAG=$(cat_gcs_file kubevirt-prow "$GCS_FILE_PATH") || { echo "Failed to fetch KUBEVIRTCI_TAG"; exit 1; } &&
+            echo "Fetched KUBEVIRTCI_TAG: $KUBEVIRTCI_TAG" &&
```

Comment:
Could we create a small script for this under hack/ ? 

A lot of the same logic is used in the centos periodic too. 

To simplify the job config?

Reply:
Maybe this doesn't make sense as it relies on a file in the being present in the bootstrap image.